src/d3a/models/strategy/external_strategies/pv.py

```diff
@@ -79,24 +84,27 @@ def _delete_offer_impl(self, arguments, response_channel):
             self.redis.publish_json(
                 response_channel,
                 {"command": "offer_delete", "status": "ready",
-                 "deleted_offers": deleted_offers})
+                 "deleted_offers": deleted_offers,
+                 "transaction_id": arguments.get("transaction_id", None)})
```

Comment:
Shouldn't `transaction_id` must be a non-none value ?

Reply:
yes, this is precaution implemented after Spyros comment: https://github.com/gridsingularity/d3a/pull/680#discussion_r395068333
If it is indeed not provided in the arguments, then the value of it would be none, which then will not be matched in the client.